### PR TITLE
NOC-94 NOC-359 migration update, display data

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**
Desktop
Career-Career-Discovery-Quizzes
Abilities-Quiz-Results-Career-Discovery-Quizzes (1)

Mobile
iPhone-15-v17-4
iPhone-15-v17-4 (1)
iPhone-15-v17-4 (2)

### CODE REVIEW

File: src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig [hunk 1]
```diff
@@ -113,12 +113,20 @@
...
+              {% else %}
+                <p class="data-item"><span class="f-d">Annual Salary</span><span class="s-d">Not available</span></p>
               {% endif %}
               {% if node.field_education_level.value is not empty %}
-                <p class="data-item"><span class="f-d">Education Level</span><span class="s-d">{{ node.field_education_level.entity.name.value }}</span></p>
+                <p class="data-item"><span class="f-d">Training, Education, Experience and Responsibilities</span><span class="s-d">{{ node.field_education_level.entity.name.value }}</span></p>
               {% endif %}
               {% if node.field_job_openings.value is not empty %}
-                <p class="data-item"><span class="f-d">Job Openings</br><span>{{ node.field_opening_from_to.value }}</span></span><span class="s-d">{{ node.field_job_openings.value|number_format(-4, ',') }}</span></p>
+                {% if node.field_job_openings.value < 0 %}
```
Comment (infojunkie) [inline src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig:123]: I'm not crazy about duplicating the template for this line just to cap to zero. Can't we do a `max(0, value)` somewhere above this line?

**Review by infojunkie — CHANGES_REQUESTED** [review]
Left a comment

File: src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig]: updated

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks

**Pull request merged**